### ISSUE TITLE
Cannot import tui-image-editor in typescript

### ISSUE THREAD

**Issue by jdamen** [nhnent/tui.image-editor#159]
## Version
npm "tui-image-editor": "3.5.1"

## Development Environment
chrome, win10

## Current Behavior
Chrome reports:

```
Uncaught TypeError: tui_image_editor_1.default is not a constructor
```

// Write example code
```js
import ImageEditor from 'tui-image-editor';


  this.imageEditor = new ImageEditor(image, {
                    cssMaxWidth: 1000, // Component default value: 1000
                    cssMaxHeight: 800  // Component default value: 800
                });

```
## Expected Behavior
image is the HTMLDivElement
it should create the image editor

However, I'm not able to instantiate the editor. 
ps. I use webpack for bundling

tsconfig:
```json
{
    "compilerOptions": {
        "module": "commonjs",
        "target": "es6",
        "lib": [ "es2018", "dom" ],
        "forceConsistentCasingInFileNames": true,
        "listEmittedFiles": true,
        "noEmitOnError": true,
        "locale": "en-US",
        "removeComments": true,
        "sourceMap": true
    },
    "exclude": [
        "node_modules"
    ]
}
```
I would be nice to have an example using typescript & webpack.

**Comment by junghwan-park** [comment]
Are you installed `tui-image-editor` correctly? Please check your `node_modules` directory. 
And you should load `tui-image-editor.js` to your HTML file when you set `tui-image-editor` to `externals` in `webpack.config.js`. Or you can bundle it single file with your codes.

**Comment by kritollm** [comment]
Hi.

I had the same issue. The reason is because the typescript declaration is wrong.

In the source code for ImageEditor
```javascript
module.exports = ImageEditor;
```

In the declarationcode for ImageEditor
```javascript
declare module 'tui-image-editor' {
    export default tuiImageEditor.ImageEditor;
}
```
Change that to
```javascript
declare module 'tui-image-editor' {
    export = tuiImageEditor.ImageEditor;
}
```

If not, the typescript compiler adds .default to the imported ImageEditor.

Also remember that in typescript you need to import export = with import = require(

[As stated here](https://www.typescriptlang.org/docs/handbook/modules.html): 
When exporting a module using export =, TypeScript-specific import module = require("module") must be used to import the module.

**Comment by junghwan-park** [comment]
@kritollm 
Oh I see. I'll check and fix right away.
cc: @jdamen

**Comment by junghwan-park** [comment]
@kritollm @jdamen 
 FYI, I just published `v3.5.2`.
